Kernel/init.cpp

```diff
@@ -68,9 +69,10 @@ VFS* vfs;
         hang();
     }
 
-    auto dev_hd0 = IDEDiskDevice::create(IDEDiskDevice::DriveType::MASTER);
+    auto pata0 = PATAChannel::create(PATAChannel::ChannelType::Primary);
+    //auto dev_hd0 = IDEDiskDevice::create(*pata0);
```

Comment:
You can just delete this. Let's not leave commented-out code around. If something is worth keeping for debugging, we can put it behind an `#ifdef FOO_DEBUG`